docs/getting_started/notifications.md

```diff
@@ -7,121 +10,132 @@ Jellyfish sends server side notifications whenever some important event occurs,
 
-```python
-from jellyfish import Notifier
+  <TabItem value="elixir" label="Elixir">
+
+```elixir
 server_address = "localhost:5002"
 server_api_token = "development"
-room_api = Notifier(server_address = server_address, server_api_token = server_api_token)
-notifier.connect()
-await notifier.wait_ready()
+
+{:ok, notifier} =
+  Jellyfish.WSNotifier.start(server_address: server_address, server_api_token: server_api_token)
+
+:ok = Jellyfish.WSNotifier.subscribe_server_notifications(notifier)
+:ok = Jellyfish.WSNotifier.subscribe_metrics(notifier)
+
+receive do
+  {:jellyfish, %Jellyfish.Notification.PeerConnected{} = notification} ->
+    IO.inspect(notification)
+  # other notificaitons ...
+end
 ```
 
+  </TabItem>
+</Tabs>
+
 As a result, you should see the following logs on the server side
+
 ```
 07:45:02.684 [info] New incoming server WebSocket connection, accepting
 07:45:02.688 [info] Server WS authenticated.
 ```
 
 ## Webhooks
-Webhook is a method of augmenting or altering the behaviour of a web page, application, or web API by allowing it to send real-time data to a specified URL when a certain event occurs.
+
+Webhooks are simply HTTP POST requests sent to a specified URL.
+They are a common choice in serverless architectures where a WebSocket connection cannot be established.
+
+The most important difference between Webhook and WebSocket notifications is that you have to receive those
+POST requests on your own and then use SDK's decode functions to get the actual notification.
 
 To receive server notifications through webhooks you have to pass `webhook_url` during room creation.
 
-#### Elixir example
+<Tabs>
+  <TabItem value="python" label="Python">
+
+```python
+from jellyfish import Notifier, RoomApi
+from jellyfish import recevie_json
 
-Example using Elixir Server SDK:
-```elixir
 server_address = "localhost:5002"
 server_api_token = "development"
-webhook_url = "http://localhost:5003/"
-
-client = Jellyfish.Client.new(server_address: server_address, server_api_token: server_api_token)
-{:ok, %Jellyfish.Room{id: room_id}, jellyfish_address} = Jellyfish.Room.create(client, max_peers: 10, webhook_url: webhook_url)
-```
+webhook_url = "http://localhost:5003/webhook"
 
-Users have to receive post requests on its own. They can decode received notificaton by passing request body to proper SDK function.
-Example using Elixir Server SDK:
-```elixir
-  def call(conn, _opts) do
-    {:ok, body, conn} = Plug.Conn.read_body(conn, [])
-    json_body = Jason.decode!(body)
+room_api = RoomApi(server_address = server_address, server_api_token = server_api_token)
+_, room = room_api.create_room(webhook_url=webhook_url)
 
-    notification = WebhookNotifier.receive(json_body)
+@app.route("/webhook", methods=["POST"])
+def respond_root():
+    json = request.get_json()
+    notification = receive_json(json)
 
-    conn
-    |> put_resp_content_type("text/plain")
-    |> send_resp(200, "OK")
-  end
+    return Response(status=200)
```

Comment:
To make it similar to the Elixir example we should mention that we assume that the user uses Flask.